data/remnant/remnant ships.txt

```diff
@@ -233,6 +233,323 @@ ship "Albatross" "Albatross (Ember Shield)"
+		"shield protection" 100
+		"slowing protection" 1
+		"discharge protection" 1
+		"corrosion protection" 100
+		"leak protection" 100
+		"burn protection" 100
+		inscrutable 1
+	outfits
+		"Fusion Reactor"
+		"LP144a Battery Pack"
+		
+		"D94-YV Shield Generator"
+		"Large Radar Jammer"
+		"Ramscoop"
+		"NDR-114 Android" 100
+		"Laser Rifle" 100
+		
+		"X4700 Ion Thruster"
+		"X5200 Ion Steering"
+		"Hyperdrive"
+
+	engine 32.5 207
+	engine 3 207
+	engine -27.5 207
+	turret 10 -136.5
+	turret -21.5 -57.5
+	turret -63.5 120
+	turret 51 101
+	turret 52.5 143
+	gun 65 -111 
+		angle 40
+	gun -72 -97.5 
+		angle 320
+	gun 78 64 
+		angle 90
+	gun -66.5 61 
+		angle 270
+		
+
+ship "Dromedary" "Dromedary Specter"
+	sprite "ship/dromedary specter"
+	thumbnail "thumbnail/dromedary wreck"
+	"uncapturable"
+	"never disabled"
+	attributes
+		licenses
+			Remnant Capital
+		category "Transport"
+		"cost" 57600000
+		"shields" 5100
+		"hull" 5600
+		"required crew" 95
+		"bunks" 300
+		"mass" 2140
+		"drag" 16.1
+		"heat dissipation" .6
+		"fuel capacity" 900
+		"cargo space" 550
+		"outfit space" 740
+		"weapon capacity" 270
+		"engine capacity" 195
+		"outfit scan power" 18
+		"outfit scan speed" 1
+		"tactical scan power" 39
+		"asteroid scan power" 42
+		"atmosphere scan" 100
+		weapon
+			"blast radius" 260
+			"shield damage" 2600
+			"hull damage" 1300
+			"hit force" 3900
+		"Automaton" 1
+		"disruption protection" 100
+		"force protection" 100
+		"fuel protection" 100
+		"hull protection" 100
+		"ion protection" 1
+		"piercing protection" 100
+		"shield protection" 100
+		"slowing protection" 1
+		"discharge protection" 1
+		"corrosion protection" 100
+		"leak protection" 100
+		"burn protection" 100
+		inscrutable 1
+	outfits
+		"Fusion Reactor"
+		"LP144a Battery Pack"
+		"D94-YV Shield Generator"
+		"Large Radar Jammer"
+		"Ramscoop"
+		"NDR-114 Android" 100
+		"Laser Rifle" 15
+		
+		"X4700 Ion Thruster"
+		"X5200 Ion Steering"
+		"Hyperdrive"
+
+	engine 32.5 207
+	engine 3 207
+	engine -27.5 207
+	turret 10 -136.5
+	turret -21.5 -57.5
+	turret -63.5 120
+	turret 51 101
+	turret 52.5 143
+	gun 65 -111 
+		angle 40
+	gun -72 -97.5 
+		angle 320
+	gun 78 64 
+		angle 90
+	gun -66.5 61 
+		angle 270
+
+
+ship "Dromedary" "Dromedary Wreck"
+	sprite "ship/dromedary wreck"
+	thumbnail "thumbnail/dromedary wreck"
+	attributes
+		licenses
+			Remnant Capital
+		category "Transport"
+		"cost" 17600000
+		"shields" 5100
+		"hull" 5600
+		"required crew" 95
+		"crew" 13
+		"bunks" 300
+		"mass" 2140
+		"drag" 16.1
+		"heat dissipation" .6
+		"fuel capacity" 900
+		"cargo space" 550
+		"outfit space" 740
+		"quantum keystone" 1
+		"weapon capacity" 270
+		"engine capacity" 195
+		"outfit scan power" 15
+		"outfit scan speed" 1
+		"tactical scan power" 25
+		"asteroid scan power" 42
+		"atmosphere scan" 100
+		"hull protection" 1
+		"ion protection" 1
+		"slowing protection" 1
+		"discharge protection" 1
+		"burn protection" 100
```

Comment:
Is it intended it only has 5000 shields? (didn't you forget a 1 there? it feels really low)
I'm trying to understand how this works, but I think it should be made clear somewhere in this PR: what does this hull protection do? 1% less damage?
I would love the remnants to make this ship unique, say "we managed to put this ship partially out of phase, as was the original", and give it 10%, 25% resistance if that's what this is? Given there is no way to increase shields it  does not seem like much of a balance issue but it would give one a reason to actually use those ships ever without instantly getting blown to bits?

Reply:
Terms on the wreck variant. I used those resistance (and the absurdly high ones on the other variants) to represent various situational effects. Such as not being all there, or in the case of the wreck there is vast amount of wreckage absorbing shots.

The shield *are* supposed to be that low on the variants. This is a ship running 500 year old tech, and even that has been damaged. This is intended to be kind of a window into the past. 500 years ago, having 5k Shields was pretty good. Maybe not quite heavy warship grade, but still on the high end of things.